packages/paste-core/core-bundle/package.json

```diff
@@ -35,6 +35,7 @@
     "@twilio-paste/button": "^1.1.45",
     "@twilio-paste/card": "^1.3.30",
     "@twilio-paste/combobox-primitive": "^0.1.0",
+    "@twilio-paste/disclosure-primitive": "^0.0.2",
```

Comment:
You need to add `@twilio-paste/reakit-library` as a peer dependency to core. Right now it’s bundling it into core, hence the size increase of 3kb or so. If it’s a peer you should see that size drop.

Eventually we’d create a bundle like core, for the libraries so we don’t need to do this for every library package.